modules/ROOT/pages/access-developer-portal-task.adoc

```diff
@@ -1,27 +1,34 @@
-= Accessing the Developer Portal
+= Access the Developer Portal
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-API portals listed on an organization's developer portal that are public, or to which you have been granted Portals Viewer access, are visible to you. From the public Anypoint Platform Developer portal, you can access Anypoint Platform APIs to automate Anypoint Platform activities.
+As an administrator, you can view API portals if you have the appropriate access. The public https://anypoint.mulesoft.com/exchange/portals/anypoint-platform/[Anypoint Platform developer portal], enables you to automate Anypoint Platform activities.
+ 
+You can view an API portal in Anypoint Platform if one of the below conditions is true:
 
-In API Manager, from the API Administration page, use the hamburger menu on the right. Alternatively, use the following procedure.
+* The portal is listed in the organization's public developer portal
```

Comment:
```suggestion
* The portal is public.
```

Reply:
How would they know where to check if it is public or not. I have to tell them where.